spec/validators/isbn_validator_spec.rb

```diff
@@ -0,0 +1,78 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+RSpec.describe IsbnValidator do
+  let(:model_class) do
+    Class.new do
+      include ActiveModel::Model
+      attr_accessor :isbn
+    end
+  end
+
+  let(:model) { model_class.new }
+
+  let(:valid_value) { "978-3-16-148410-0" }
```

Comment:
We have `Faker` that allows us to randomize this.

```suggestion
  let(:valid_value) { Faker::Code.isbn(base: 13) }
```